pyproject.toml

```diff
@@ -121,6 +109,7 @@ exclude = [
   "docs",
   "htmlcov",
   "node_modules",
+  "outputs",
```

Comment:
And `static`?

Reply:
Unless I've missed something, `static` is already included below.

`outputs` was the only entry for the Black excludes, that wasn't listed already in Ruff's excludes.